Makefile

```diff
@@ -1,3 +1,15 @@
+# Git Submodules
+SUBMODULES = \
+	gbit 
+
+GIT_SUBMODULES := $(shell git submodule update --init $(SUBMODULES))
```

Comment:
If `gbit` exists and is valid, you should ignore the above command. Check jitboy on the way to validate the git sub-module.

Reply:
like this?
```bash
GBIT_LIB_C := \
	gbit/lib/tester.c \
	gbit/lib/ref_cpu.c \
	gbit/lib/inputstate.c \
	gbit/lib/disassembler.c

$(GBIT_LIB_C):
	git submodule update --init
	touch $@
```